src/components/plots/plot.ts

```diff
@@ -266,22 +276,23 @@ module Plottable {
      * Updates the extents associated with each attribute, then autodomains all scales the Plot uses.
      */
     protected _updateExtents() {
-      this._attrBindings.forEach((attr) => this._updateExtentsForAttr(attr));
+      this._attrBindings.forEach((attr) => this._updateExtentsForKey(attr, true));
+      this._propertyExtents.forEach((property) => this._updateExtentsForKey(property, false));
       this._scales().forEach((scale) => scale._autoDomainIfAutomaticMode());
     }
 
-    private _updateExtentsForAttr(attr: string) {
-      var binding = this._attrBindings.get(attr);
-      var accessor = binding.accessor;
-      var scale = binding.scale;
-      var coercer = (scale != null) ? scale._typeCoercer : (d: any) => d;
-      var extents = this._datasetKeysInOrder.map((key) => {
+    protected _updateExtentsForKey(key: string, ifAttr: boolean) {
```

Comment:
Invocations of this method might be more readable if you passed in the `d3.Map` instead of a boolean.